### PR TITLE
Cover and itemList - react version

### PR DESCRIPTION
**Pull request by rageofaxe**
https://github.com/webRunes/Default-WRIO-Theme/issues/87

### CODE REVIEW

**Comment by AlexeyAnshakov** [comment]
@spirinvladimir please review

**Comment by AlexeyAnshakov** [comment]
@spirinvladimir please review again

**Comment by spirinvladimir** [comment]
JavaScript source code should be lintable. We could use eslint and specify all rules we like for this project

**Comment by AlexeyAnshakov** [comment]
@spirinvladimir https://github.com/webRunes/WRIO-InternetOS/issues/231

**Comment by AlexeyAnshakov** [comment]
@spirinvladimir please review

**Comment by spirinvladimir** [comment]
At existing code was some eslint erros. I solved them at [commit](https://github.com/webRunes/WRIO-InternetOS/commit/389f0b97a5f5a91597449c76662c84da3eb16743). Please merge your code with my commit.

Your last [commit](https://github.com/rageofaxe/WRIO-InternetOS/commit/2788d24b5ae05e64d43b77943e037f6f2e6c7cc5) has conflicts: 

Merge remote-tracking branch 'orig/master'
Conflicts:
    WRIO-InternetOS/js/components/CreateArticleElement.js
    WRIO-InternetOS/js/jsonld/scripts.js
    WRIO-InternetOS/js/mixins/mentions.js
    package.json

File: WRIO-InternetOS/js/addBootstrapLink.js [hunk 1]
```diff
@@ -26,4 +26,15 @@ module.exports = function (cb) {
...
+
+    var js = [];
+
+
+    js.forEach(function (src) {
+        var script;
+        script = document.createElement('script');
+        script.setAttribute('type', 'text/javascript');
+        script.setAttribute('src', src);
+        document.body.appendChild(script);
```
Comment (spirinvladimir) [inline WRIO-InternetOS/js/addBootstrapLink.js]: Better append script to head. DOM is already loaded.

File: WRIO-InternetOS/js/mixins/mentions.js [hunk 2]
```diff
@@ -15,24 +15,24 @@ module.exports = {
             intersection = function (s1, s2) {
                 s2 = s2 || '';
                 var a1 = s1.split('').reverse(),
-                    a2 = s2.split(''),
-                    j = 0;
-                while (a1[j] && a2[j] && (a1[j] === a2[j])) {
-                    j += 1;
+                    a2 = s2.split('');
```
Comment (spirinvladimir) [inline WRIO-InternetOS/js/mixins/mentions.js]: Don't change this file

**Comment by AlexeyAnshakov** [comment]
@spirinvladimir please review

**Comment by AlexeyAnshakov** [comment]
@spirinvladimir merged but review for next changes

**Pull request merged**